.github/workflows/ci.yml

```diff
@@ -21,66 +21,70 @@ jobs:
           -DCMAKE_BUILD_TYPE=Debug \
           -DCMAKE_C_FLAGS='--coverage -Wall -Wextra -DNDEBUG'
           make -C build/ all
-      - name: Test
-        run: |
-          cd build/
-          ctest -E system --output-on-failure
-          cd ..
+
+      - name: Run CTests
+        run: ctest --test-dir build -E system --output-on-failure
+
       - name: Run Coverage
         run: |
           make -C build/ coverage
           declare -a EXCLUDE=("\*test\*" "\*CMakeCCompilerId\*" "\*mocks\*" "\*3rdparty\*")
           echo ${EXCLUDE[@]} | xargs lcov --rc lcov_branch_coverage=1 -r build/coverage.info -o build/coverage.info
           lcov --rc lcov_branch_coverage=1 --list build/coverage.info
       - name: Check Coverage
-        uses: FreeRTOS/CI-CD-Github-Actions/coverage-cop@main
+        uses: FreeRTOS/CI-CD-Github-Actions/coverage-cop@v2
         with:
-          path: ./build/coverage.info
+          coverage-file: ./build/coverage.info
+
   complexity:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Check complexity
-        uses: FreeRTOS/CI-CD-Github-Actions/complexity@main
+        uses: FreeRTOS/CI-CD-Github-Actions/complexity@v2
         with:
           path: ./
+
   doxygen:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Run doxygen build
-        uses: FreeRTOS/CI-CD-Github-Actions/doxygen@main
+        uses: FreeRTOS/CI-CD-Github-Actions/doxygen@v2
         with:
           path: ./
+
   spell-check:
     runs-on: ubuntu-latest
     steps:
       - name: Clone This Repo
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
       - name: Run spellings check
-        uses: FreeRTOS/CI-CD-Github-Actions/spellings@main
+        uses: FreeRTOS/CI-CD-GitHub-Actions/spellings@v2
         with:
           path: ./
+
   formatting:
     runs-on: ubuntu-20.04
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Check formatting
-        uses: FreeRTOS/CI-CD-Github-Actions/formatting@main
+        uses: FreeRTOS/CI-CD-Github-Actions/formatting@v2
         with:
           path: ./
+
   ssot-check:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout this repo
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
           path: current
       - name: Checkout coreMQTT
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
-          ref: main
-          repository: FreeRTOS/coreMQTT
+          ref: CI-CD-Updates
+          repository: Skptak/coreMQTT
```

Comment:
This should be updated to FreeRTOS/coreMQTT, correct?

Reply:
Changed in formatting has caused the header to change, as such need to point to my version of the file to pass the check.